examples/event-sources/generic.yaml

```diff
@@ -0,0 +1,14 @@
+apiVersion: argoproj.io/v1alpha1
+kind: EventSource
+metadata:
+  name: generic
+spec:
+  generic:
+    example:
+      insecure: true
+      # URL of the gRPC server that implements event source
+      url: "generic-event-source-server.argo-events.svc:8080"
+      # Config represents the configuration required to run the event source
+      config: |-
```

Comment:
Not quite sure how to use `config`, expecting a document to explain how to use `GenericEventSource`.

Reply:
Added docs on generic es

Reply:
added docs